### PR TITLE
Streamline envelope action menus

### PR DESCRIPTION
**Pull request by StCyr**
Fixes https://github.com/nextcloud/mail/issues/4256

Signed-off-by: Cyrille Bollu <cyrpub@bollu.be>

### CODE REVIEW

File: src/components/Envelope.vue [hunk 1]
```diff
@@ -58,103 +58,32 @@
...
-		<MoveModal v-if="showMoveModal"
-			:account="account"
-			:envelopes="[data]"
-			@close="onCloseMoveModal" />
+		<MenuEnvelope
+			:envelope=data
+			:mailbox=mailbox
+			:isSelected=selected
+			:can-select=true
+			:with-show-source=false
```
Comment (ChristophWurst) [inline src/components/Envelope.vue:66]: can-show-source?

File: src/components/MenuEnvelope.vue [hunk 2]
```diff
@@ -0,0 +1,321 @@
+<!-- Standard Actions menu for Envelopes -->
+<template>
+	<div class="app-content-list-item-menu">
```
Comment (ChristophWurst) [inline src/components/MenuEnvelope.vue:3]: I'm pretty sure this class should only be used in the *app content list* (envelope list) and no in the details on the right

File: src/components/MenuEnvelope.vue [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+			this.showMoveModal = true
+		},
+		onCloseMoveModal() {
+			this.showMoveModal = false
+		},
+	},
+}
+</script>
+
+<style>
```
Comment (ChristophWurst) [inline src/components/MenuEnvelope.vue:308]: lang=scss?

**Review by ChristophWurst — COMMENTED** [review]
Great move!

some minor comments. also eslint is your friend, let is fix some syntax issues :)

File: src/components/Envelope.vue (hunk 1 above)
Comment (StCyr) [reply src/components/Envelope.vue]: with-select 😉

**Comment by GretaD** [comment]
All looks pretty good. The only thing i noticed that was shown wrong, is the account color. I have 2 accounts, but the colour at the messaged(second column) is always pink. On master is all good.

rect825

**Comment by StCyr** [comment]
> 
> All looks pretty good. The only thing i noticed that was shown wrong, is the account color. I have 2 accounts, but the colour at the messages(second column) is always pink. On master is all good.
> 

Hey! Good catch!

It's fixed now

**Review by GretaD — APPROVED** [review]
all looks good. Tested and it works

**Pull request merged**